views/js/qtiCreator/plugins/menu/preview.js

```diff
@@ -59,7 +58,7 @@ define([
             //wrap the preview opening
             //TODO move away the URLs !!!
             var openPreview = function openPreview(){
-                preview.init(helpers._url('index', 'QtiPreview', 'taoQtiItem', { uri : item.data('uri') }));
+                preview.init(itemCreator.getConfig().properties.previewUrl + '?uri=' + encodeURIComponent(item.data('uri')));
```

Comment:
The `previewUrl` can have already parameters. Please use instead the `build` function of `util/url`